Environment/Environment.pillar

```diff
@@ -60,15 +59,18 @@ central place compared to debuggers for other programming languages, because in
 Pharo you can ''program'' in the debugger. The debugger is not
 launched from a menu; it is normally entered by running a failing test, by
 typing ==CMD-.== to interrupt a running process, or by inserting a
-==self halt== expression in code.
+==Halt now== expression in code.
```

Comment:
Damien Cassou told me that "self halt"  still exist but it's better to use "Halt now".